### ISSUE TITLE
Hubspot oauth

### ISSUE THREAD

**Issue by vitorallo** [n8n-io/n8n-docs#2594]
the documentation is missing one scope, took several attempts until I debugged the request. 
this is the current scope requested authenticating from the n8n Hubspot connector to the app.

scope: crm.lists.write crm.objects.contacts.read crm.objects.contacts.write crm.objects.companies.read crm.objects.companies.write crm.objects.deals.read crm.objects.deals.write crm.objects.owners.read crm.schemas.companies.read crm.schemas.contacts.read crm.schemas.deals.read forms tickets

as you can see, the crm.lists.write is not included in the documentation available here: https://docs.n8n.io/integrations/builtin/credentials/hubspot/?utm_source=n8n_app&utm_medium=credential_settings&utm_campaign=create_new_credentials_modal#required-scopes-for-hubspot-trigger-node

so long story short, you'll never authenticated. I am not sure why the lists.write scope is requested, it might be a problem of my specific version of the Hubspot component, didn't get into the code to understand so my debugging stops here. just reporting it, for a fix in the documentation or a fix to the code as needed

**Comment by Joffcom** [comment]
Hey @vitorallo,

This has now been updated, thanks for the reminder. We use the scope because we support some of the list options.
